src/proxy-ui-api/frontend/src/components/client/SelectClientDialog.vue

```diff
@@ -24,7 +24,7 @@
    THE SOFTWARE.
  -->
 <template>
-  <v-dialog :value="dialog" width="750" scrollable persistent>
+  <v-dialog v-if="dialog" :value="dialog" width="750" scrollable persistent>
```

Comment:
A bit theoretical note but is there any benefit keeping :value="dialog"  in these? I guess it works just similarly without them.